prd.md

```diff
@@ -0,0 +1,119 @@
+# Co-Hosting Feature PRD
+
+## Overview
+
+Add support for multiple hosts (co-hosts) to events in Shera, where all hosts have equal permissions and capabilities.
+
+## Feature Details
+
+### Core Functionality
+
+- All hosts have equal permissions and capabilities
+- No limit on number of co-hosts per event
+- Cannot remove the last host from an event
+
+### User Interface
+
+1. Edit Event Page Updates:
+
+   - Add "Hosts" section
+   - Display list of current hosts with remove buttons
+   - Add "Invite Co-host" button
+   - Use existing invite component (network/email/link) for co-host invitations
+
+2. Host Display Format:
+   - For 3 or fewer hosts: "Hosted by [name1], [name2] and [name3]"
+   - For more than 3 hosts: "Hosted by [name1] and X others"
+
+### Technical Implementation
+
+#### Database Changes
+
+1. Modify Event model in schema.prisma:
+
+```prisma
+model Event {
+    // ... existing fields ...
+    hosts     User[]    @relation("EventHosts")
```

Comment:
Thinking a bit forward: Does a host need to be a User?
Can for instance an organization/company be the host of an event?

I actually tried thinking 10 minutes on this problem yesterday, and introduced 



```prisma
model EventHost{
    contactUser    User
    type           EventHostType
    privilege      HostPrivilege
}

enum EventHostType{
    User
    Organization
    // Other
}

enum HostPrivilege{
    MAIN_HOST
    CO_HOST
}
```

Reply:
Organizations hosting events is for sure something I'm thinking about, but I'm not convinced that we still wouldn't have the user model as the host in that use case. It seems a lot harder to have to consider organizations vs users in all interactions with the host than just make it easy from the UI to add other organization members as hosts to an event. 

In any case we don't have organizations right now and developing any meaningful functionality for that is beyond the scope of this PR. For sure something to keep thinking about, feel free to share some thoughts in #106